lib/src/app-switcher/app-switcher.component.scss

```diff
@@ -1,4 +1,4 @@
-@import '../colors.scss';
+@import '../sass/cashmere';
```

Comment:
there is going to be quite a bit of style duplication if every component imports all the cashmere styles are imported for every component. We can import variables and mixins because they dont actually create any styles, they are just functional. 

We can discuss it, but I would say we keep colors and give instructions on how to include the cashmere styles (or parts of it) in the main readme